.agent-versions.json

```diff
@@ -4,8 +4,6 @@
     "8.15.0",
     "8.14.4-SNAPSHOT",
     "8.14.3",
-    "8.13.4",
-    "7.17.24-SNAPSHOT",
-    "7.17.23"
```

Comment:
Is just removing 7.17.23 enough to get past this? Or is it failing for both 7.17.x versions?

Reply:
Answer on Slack, it is failing for all of them.